Tests/LinuxMain.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
```

Comment:
Why is this back?

Reply:
Because I forgot about test discovery :) I'll remove it.